cms/drupal/db/backup/import.php

```diff
@@ -17,8 +17,23 @@
 
 task('cms:drupal:db:backup:import', static function (): void {
     $latestBackup = runLocally('ls -tr -1 {{local_database_backups}} | tail -1');
-    VirtualMachine::run('drush -r {{app_directory_name}} rq && drush sql-drop');
-    VirtualMachine::run('gunzip -c {{local_database_backups}}/' . $latestBackup . '| drush -r {{app_directory_name}} sql:cli');
-    VirtualMachine::run('drush -r {{app_directory_name}} cr');
+    VirtualMachine::run('drush sql-drop -y');
+    runLocally('gzip -dfq {{local_database_backups}}/' . $latestBackup);
+    \sleep(10); // Sleep 10 seconds waiting for container sync to catch up.
+    $unzipped = VirtualMachine::run('ls -tr -1 {{local_database_backups}} | tail -1');
+    /** SET commands that are exported (mysqldump) from some mysql environments
+        (often those coming from master slave environments ) do not import
+        on single mysql containers and throw this error:
+     *
+     *  > ERROR 1227 (42000) at line 18: Access denied; you need (at least one
+     *  of) the SUPER, SYSTEM_VARIABLES_ADMIN or SESSION_VARIABLES_ADMIN
+     *  privilege(s) for this operation
+     *
+     *  It is harmless to strip these out for development purposes only.
```

Comment:
While I am inclined to agree that it is harmless to remove such commands from the SQL file for development purposes only, there remains a very real risk that a user will migrate a database imported to a VM back to a production environment that requires such SET commands. In my opinion this package should not modify database information. If we decide otherwise, it should be prominently noted in the README.
It seems this particular issue might be best solved by ensuring local & remote environment configurations match (which would fall outside the scope of this package).
This package could also provide a task which includes such data transformation, but is _optional_ by default (e.g. it is manually executed and/or executed on a project-by-project basis as part of a `before`/`after` hook).
Please let me know your thoughts on the matter. Thanks!

Reply:
Ok sounds good. closing for another time and method.

Reply:
Thanks @danshumaker!